spec/order_total_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'order_total'
+
+describe OrderTotal do
+
+  let(:burger_description) {"Cheeseburger"}
+  let(:burger_price) {4.99}
+  let(:burger_to_s) {"#{burger_description}\t#{burger_price}"}
+  let(:burger) { double :dish, :description => burger_description,
+                              :price => burger_price,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)